src/components/Layout/Seo/Seo.tsx

```diff
@@ -51,6 +54,8 @@ export const Seo: FC<SeoProps> = ({
   const metaImage = image ?? site.siteMetadata?.image;
   const keywords = site.siteMetadata?.keywords;
   const metaSiteName = site.siteMetadata?.siteName;
+  const previewImageUrl =
+    (previewImage && `https:${previewImage}`) || site.siteMetadata?.previewImage;
```

Comment:
```suggestion
    previewImage ? `https:${previewImage}`) : site.siteMetadata?.previewImage;
```
Also please leave a comment why we need such an addition with `https:`.